src/ImageReconstruction.jl

```diff
@@ -74,29 +73,42 @@ function iradon(sinogram::AbstractMatrix, θ::AbstractRange, t::AbstractRange)
     K = length(θ)
     Npad = nextpow(2, 2 * N - 1)
     τ = step(t)
-    ramp = fft(_zero_pad(_ramp_spatial(N, τ), Npad - N))
+    ramp = fft(_ramp_spatial(N, τ, Npad))
     i = div(N, 2) + 1
     j = i + N - 1
 
-    image = zeros(eltype(sinogram), pixels, pixels)
-    Q = Vector{eltype(sinogram)}(undef, N)
+	T = eltype(sinogram)
+	image = [zeros(T, pixels, pixels) for _ = 1:nthreads()]
+	P′ = [Vector{Complex{T}}(undef, Npad) for _ in 1:nthreads()]
+	Q = [Vector{T}(undef, N) for _ in 1:nthreads()]
+
+	l = SpinLock()
+	@inbounds @threads for (k, θₖ) in collect(enumerate(θ))
+		id = threadid()
 
-    for (k, θₖ) in enumerate(θ)
         # filter projection
-        Q[:] .=
-            τ .*
-            real.(ifft(fft(_zero_pad(view(sinogram, :, k), Npad - N)) .* ramp)[i:j])
-        Qₖ = LinearInterpolation(t, Q)
+		P′[id][1:N] .= sinogram[:, k]
+		P′[id][N+1:end] .= 0
+
+		lock(l) # threading issues with FFTW (#134)?
```

Comment:
#134 doesn't point to any issue 😂

Reply:
https://github.com/JuliaMath/FFTW.jl/issues/134

Reply:
Yeah.. if it's not an issue of this package, it's better to provide an complete link.

Reply:
That's fair.